Makefile

```diff
@@ -21,7 +21,7 @@ TIMEOUT = 360
 
 .PHONY: all
 all: fmt vendor lint vet megacheck | $(BASE) ; $(info $(M) building library…) @ ## Build program
-	$Q cd $(BASE) && $(GO) build \
+	$Q cd $(BASE) && $(GO) build ./... \
```

Comment:
Adding this, because I believe its omission allowed #27 to pass CI without noticing a build failure for `dep` users.